src/pages/home/report/ContextMenu/PopoverReportActionContextMenu.js

```diff
@@ -88,6 +93,18 @@ class PopoverReportActionContextMenu extends React.Component {
         });
     }
 
+    /**
+     * Returns full link address to show under the title on native platforms and mWeb
+     * @returns {String}
+     */
+    getPopoverDescription() {
+        const isNative = Platform.OS === 'ios' || Platform.OS === 'android';
```

Comment:
This is an incorrect way to do platform detection. You can read more about that here: https://github.com/Expensify/App#platform-specific-file-extensions

Reply:
> This is an incorrect way to do platform detection. You can read more about that here: https://github.com/Expensify/App#platform-specific-file-extensions


I'm unsure on how to divide this logic into platform-specific files.
How about implementing `isNative()` method in [getOperatingSystem](https://github.com/Expensify/App/tree/main/src/libs/getOperatingSystem), which in  [index.native.js](https://github.com/Expensify/App/blob/main/src/libs/getOperatingSystem/index.native.js) would just return true and [index.js](https://github.com/Expensify/App/blob/main/src/libs/getOperatingSystem/index.js) would return false

> I uploaded a video of desktop to the PR description for you.

Thanks!

Reply:
You could take a look at some of the existing platform-specific files to get some ideas, but without digging too deep into this code, my initial ideas are:

- Create a module like `src/pages/home/report/ContextMenu/ContextMenuUtils/index.js` which would have a method that accepts `this.state.type` and `this.state.selection` as parameters.
- Create a new platform-specific component that wraps `<BaseReportActionContextMenu>` and provides the necessary logic